.env

```diff
@@ -25,3 +25,9 @@ EXCLUDE_FROM_LB_WAIT_TIME_IN_SEC=0
 # Note: the "unlimited" option does not work well with `EXCLUDE_FROM_LB_WAIT_TIME_IN_SEC`
 # option, which could potential exclude all nodes from load balancer at the same time.
 MAX_CONCURRENT_UPDATE=1
+
+# Prevent update time window setting
+# Brupop will prevent node updates within time window
+# when you set up time window start and end time, you should use UTC (24-hour time notation).
+PREVENT_UPDATE_SINCE=0:0:0
+PREVENT_UPDATE_UNTIL=0:0:0
```

Comment:
@webern you think it's better to rename or you think use update window logic instead of prevent window logic?

Reply:
It seems more cognitively intuitive to me since I am familiar with the idea of a "maintenance window", but not a "no maintenance window".

Reply:
I see, that's a good point! 
I'd like to keep "no maintenance window" because actually this feature was asked by customers when we was implementing brupop 0.1.0. Customer wanted us to provide a feature which can prevent update on current time. Therefore, I implemented on this way.

I like maintenance window this idea, if you think it worths to do in this way, I can update it! thanks : )

Reply:
Hmmm Matt brings up a good point: I'd be surprised if update windows were extremely wide leading to the need for "preventative, no maintenance" windows. My _assumption_ is that most people running kubernetes clusters will have small windows of time when updates _should_ be allowed.

So, that's a +1 from me for `UPDATE_WINDOW_START` and `UPDATE_WINDOW_END`